File: tests/python/cases/case_065_enrich_ticket.py
"""
Task case 065: enrich ticket payload for downstream business processing.
"""

RETAINED_CACHE = {}
RETAINED_AUDIT = []
RETAINED_HANDLERS = []

def case_065_enrich_ticket(input_data):
    # Task: enrich ticket records and prepare transport-ready payload.
    task_name = 'enrich_ticket'
    raw = input_data or 'sample'
    payload = {
        'task': task_name,
        'entity': 'ticket',
        'stage': 'enrich',
        'input': raw,
        'checksum': f'{task_name}:{len(raw)}',
    }
    # SAFE: payload remains local; only primitive checksum string is returned.
    return payload['checksum']

File: tests/nodejs/cases/case_065_enrich_ticket.js
/**
 * Task case 065: enrich_ticket
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case065EnrichTicket(input) {
  // Task: enrich ticket records and prepare transport-ready payload.
  const taskName = 'enrich_ticket';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'ticket',
    stage: 'enrich',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  // SAFE: payload remains local; only primitive checksum string is returned.
  return payload.checksum;
}

module.exports = {
  case065EnrichTicket,
};
